src/layouts/SideDrawer.tsx

```diff
@@ -45,6 +46,28 @@ export const SideDrawer: FC = () => {
             {t("navigation.settings")}
           </span>
         </NavLink>
+
+        {/* Beginning of the documentation icon feature */}
```

Comment:
Same here
- Comments can be removed
- Please use a translation

I suggest to use the `BookOpenIcon` here as well. No need for that special image including a fallback. In this way it will fit in with the other links which are just using icons as well.